recipe/meta.yaml

```diff
@@ -66,6 +69,8 @@ test:
     - echo %CUDA_HOME%                            # [win]
```

Comment:
It seems something has changed between last rebuild and today, so that `CUDA_HOME` is no longer defined at the test stage... 😞